### DISCUSSION TITLE
Ventura Guidance Revision 3.0

### DISCUSSION THREAD

**Discussion by robertgendler** [usnistgov/macos_security#303] [General]
Included in this release are updated guidance documents (HTML, PDF, XLS, SCAP) for the NIST SP 800-53r5 Low, Moderate, and High, NIST 800-171, DISA-STIG, CNSSI-1253 Low, Moderate, and High, CMMC Level 1, CMMC Level 2, CIS macOS Benchmarks Level 1 and 2, and CIS Critical Security Controls Version 8 baselines for macOS Ventura (13.0).

<hr /><em>This discussion was created from the release <a href='https://github.com/usnistgov/macos_security/releases/tag/ventura_rev3'>Ventura Guidance Revision 3.0</a>.</em>

**Comment by ghost** [comment]
Looks really great.... I'm loving the sshd disable and the new pmset.... I did have a question about why not give guidance for the entire pmset variables, since sleep and displaysledp can be configured as well. Additionally, I am seeing some "granularity" issues with disabling iCloud services... and I wonder if there is a layered/hierarchical filtering on policies variables if a higher service is enabled. In my case I am using Documents and Desktop syncing, along with Keychain... but I can disable Mail and Notes but not Calendar /Reminders/Contacts. 

Is there guidance to see error logs for why some rules get applied and others don't? And by logs I mean on the system-side... there's not much on the MDM dashboard and I trolled through the system logs in /var and /Library/Logs and I can't find anything. 

<img width="774" alt="Screenshot 2023-09-24 at 4 27 37 PM" src="https://github.com/usnistgov/macos_security/assets/102542366/21e72f49-8b2c-4d08-b85e-970c2ae60f96">
<img width="486" alt="Screenshot 2023-09-24 at 4 27 23 PM" src="https://github.com/usnistgov/macos_security/assets/102542366/31b54f77-ef12-418e-b7bc-7f7bf78486d2">
